pytorch_lightning/trainer/connectors/logger_connector/metrics_holder.py

```diff
@@ -0,0 +1,80 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
shall this rather go to metric sub-package?

Reply:
I think this is fine here since this is specifically for logging and metrics shall not be entangled to the rest of lightning

Reply:
It isn't related to Metric. It is just a Dictionary Holder for LoggerConnector.

Reply:
Could change the class name.